rules/group.go

```diff
@@ -684,6 +685,7 @@ func (g *Group) RestoreForState(ts time.Time) {
 		// No results for this alert rule.
 		if len(seriesByLabels) == 0 {
 			level.Debug(g.logger).Log("msg", "Failed to find a series to restore the 'for' state", labels.AlertName, alertRule.Name())
```

Comment:
The message says "failed" but I'm not sure we actually failed. What happened here is that we haven't found any series, but it may be legit (e.g. they weren't defined before). Maybe we could change the message to something like "No series found ..."?

Reply:
Good shout, done in https://github.com/prometheus/prometheus/pull/14048/commits/dcdd851921ff6663ad4b1d6013bfe0f2223ba5ce